### PR TITLE
Sc 15476/refactor get endpoints on schools router

### PR DESCRIPTION
**Pull request by GilaHB**

### CODE REVIEW

File: neomodel/match.py [hunk 1]
```diff
@@ -322,7 +324,13 @@ def build_traversal(self, traversal):
         # build source
         lhs_ident = self.build_source(traversal.source)
         rhs_ident = traversal.name + rhs_label
-        self._ast['return'] = traversal.name
+        # self._ast['return'] = traversal.name
```
Comment (yitzc-classe) [inline neomodel/match.py]: Delete commented line

File: neomodel/match.py [hunk 2]
```diff
@@ -496,16 +504,16 @@ def _execute(self, lazy=False):
             # inject id = into ast
             self._ast['return'] = 'id({})'.format(self._ast['return'])
         query = self.build_query()
-        results, _ = db.cypher_query(query, self._query_params, resolve_objects=True)            
+        results, _ = db.cypher_query(query, self._query_params, resolve_objects=True)
```
Comment (yitzc-classe) [inline neomodel/match.py:505]: Not sure why this whitespace was marked as a change..?

File: neomodel/match.py [hunk 3]
```diff
@@ -734,6 +743,12 @@ def order_by(self, *props):
 
         return self
 
+    def values(self, *args):
```
Comment (yitzc-classe) [inline neomodel/match.py:744]: Docstring!

File: neomodel/relationship_manager.py [hunk 4]
```diff
@@ -387,6 +387,9 @@ def all(self):
         """
         return self._new_traversal().all()
 
+    def values(self, *args):
```
Comment (yitzc-classe) [inline neomodel/relationship_manager.py:390]: Docstring!

**Review by yitzc-classe — CHANGES_REQUESTED** [review]

File: neomodel/match.py (hunk 2 above)
Comment (GilaHB) [reply neomodel/match.py:505]: No idea. I just removed some whitespace because it was bothering my eyes but that's all it was.

**Review by yitzc-classe — APPROVED** [review]
Looks good.
One day, we'll write some tests for these additions also. 
This will make it easier to understand, for example, how to use the `values()` function. 
But for now it's fine.

**Pull request merged**